config_defaults_inc.php

```diff
@@ -1214,6 +1214,16 @@
  */
 $g_private_news_threshold = DEVELOPER;
 
+############
+# Markdown #
+############
+
+/**
+ * Enables markdown support - once enabled all issue titles, text, and notes are formatted based on
+ * markdown syntax.  Default to OFF since it is experimental feature for now.
+ */
+$g_markdown_enabled = OFF;
```

Comment:
Regarding the application global setting of markdown, i suggest that this is icluded within the coreFormatting plugin
Mantis core will not call the markdows formatter directly, but through the coreFormatting as a proxy